### ISSUE TITLE
Accessibility: cannot tab through the add to project form

### ISSUE THREAD

**Issue by benjaminapetersen** [openshift/origin-web-catalog#406]
The `tab` key should be able to move past the `advanced options` link & proceed to the `cancel` and `create` buttons.  

![2017-08-29 16 31 15](https://user-images.githubusercontent.com/280512/29842711-a27b2bc8-8cd7-11e7-8e1d-c0d3c7211406.gif)

This may be an upstream issue w/pf-wizard?

@spadgett @jeff-phillips-18

**Comment by spadgett** [comment]
Looks like it. The button is not really a button. It just has button styling and no tabindex.

<img width="945" alt="screen shot 2017-08-29 at 6 31 33 pm" src="https://user-images.githubusercontent.com/1167259/29847198-7626c594-8ce8-11e7-865f-69881e87109b.png">

**Comment by spadgett** [comment]
Close button on the dialog also doesn't have an `href`, so it's not focusable. (This one is our bug.)

<img width="696" alt="screen shot 2017-08-29 at 6 33 25 pm" src="https://user-images.githubusercontent.com/1167259/29847228-9ac70d28-8ce8-11e7-8bf6-9654037771f4.png">

**Comment by spadgett** [comment]
I noticed also you can tab outside of the dialog onto the page below when it's meant to be a modal.

**Comment by spadgett** [comment]
Fixed by https://github.com/openshift/origin-web-console/pull/2103
